src/Microsoft.SqlTools.ServiceLayer/Cms/Contracts/RegisterServerParams.cs

```diff
@@ -0,0 +1,84 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+using Microsoft.SqlTools.ServiceLayer.Connection.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.Cms.Contracts
+{
+    /// <summary>
+    /// Paramaters to create Top level Central Management Server
+    /// </summary>
+    public class CreateCentralManagementServerParams
+    {
+        public string RegisteredServerName { get; set; }
+
+        public string RegisteredServerDescription { get; set; }
+
+        public ConnectParams ConnectParams { get; set; }
+    }
+
+    /// <summary>
+    /// Parmaters to Add Registered Server to top level CMS
+    /// </summary>
+    public class AddRegisteredServerParams
+    {
+        public string RegisteredServerName { get; set; }
+
+        public string RegisteredServerDescription { get; set; }
+        
+        public ConnectionDetails RegServerConnectionDetails { get; set; }
```

Comment:
we should be consistent whether to use `RegisteredServer` or `RegServer`, personally I would always use `RegisteredServer` event though it's too verbose.